pages/common/scrcpy.md

```diff
@@ -0,0 +1,28 @@
+# scrcpy
+
+> Display and control your Android device on a desktop .
+> More information: <https://github.com/Genymobile/scrcpy>.
+
+- Display/mirror your device:
```

Comment:
I think we should choose 1 verb here. Maybe @sbrl knows how to write it.

Reply:
Ooh, this looks like an interesting one. What about this?
```suggestion
- Display a mirror of a connected device:
```

We generally try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr pages.

Reply:
This is a scenario for `cast`?
```suggestion
- Cast an Android device:
```

Reply:
Not sure there. I think we need additional wording to make `cast` more clear if we went with that option